assets/src/scss/blocks/Accordion/AccordionStyle.scss

```diff
@@ -49,7 +49,7 @@
       font-size: $font-size-sm;
       border-radius: 4px;
       font-weight: var(--font-weight-regular);
-      line-height: 1;
+      line-height: var(--font-size-m--font-family-primary);
```

Comment:
The token you've assigned refers to `font-size` but you might use the ones related to `line-height` , i.e. `--line-height-{size}--{font-family}`.

I can suspect that [`--line-height-xs--font-family-primary`](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/base/_tokens.scss#L19) is the proper one here, but let confirm with the design with the design team, please!

Reply:
Fixed!